gunicorn/arbiter.py

```diff
@@ -494,6 +496,11 @@ def manage_workers(self):
 
     def spawn_worker(self):
         self.worker_age += 1
+        worker_id = -1
+        for i in range(self.num_workers):
```

Comment:
Thanks for all the feedback, @benoitc.

Despite the added overhead, I think one of the most useful parts of instance identification is having the instance number as a basic counter in a predictable range, as opposed to a hash or other unique identifier. If implementing this with a hash it wouldn't really add much to gunicorn, as each worker's PID can already act as such a unique identifier.

The use cases for this are plentiful, but could include workers logging under the same set of names every time an application is run with gunciorn (app0, app1, etc), or having workers listen on the same set of incremental ports (1000, 1001, etc). I think the predictability that a basic counter provides is pretty important here.